src/node/coinstats.cpp

```diff
@@ -117,9 +138,17 @@ static void PrepareHash(CHashWriter& ss, const CCoinsStats& stats)
     ss << stats.hashBlock;
 }
 static void PrepareHash(std::nullptr_t, CCoinsStats& stats) {}
+// MuHash does not need the prepare step
+static void PrepareHash(MuHash3072& muhash, CCoinsStats& stats) {}
 
 static void FinalizeHash(CHashWriter& ss, CCoinsStats& stats)
 {
-    stats.hashSerialized = ss.GetHash();
+    stats.hashSerialized = ss.GetSHA256();
```

Comment:
This changes the result of `gettxoutsetinfo hash_serialized_2`

Reply:
Huh, no idea how that happened, makes no sense at all, especially inside that commit. Fixed.